src/main/java/com/atlassian/bitbucket/jenkins/internal/provider/InstanceIdentityProvider.java

```diff
@@ -0,0 +1,13 @@
+package com.atlassian.bitbucket.jenkins.internal.provider;
+
+import org.jenkinsci.main.modules.instance_identity.InstanceIdentity;
+
+public interface InstanceIdentityProvider {
+
+    /**
+     * Returns the result of calling InstanceIdentity.get();
```

Comment:
This javadoc seems to leak implementation details of the default implementation. For example, this statement wouldn't be true for a test implementation that returns a mock `InstanceIdentity`.

Reply:
Fair. I might just remove the body entirely.